accord-core/src/main/java/accord/local/cfk/CommandsForKey.java

```diff
@@ -1603,35 +1603,33 @@ private CommandsForKeyUpdate update(InternalStatus newStatus, Command updated, @
         {
             // update
             TxnInfo cur = byId[pos];
-            if (cur != null)
```

Comment:
for me: 

`pos >= 0` so its known that the field isn't null, so this just removes the `null` check; i couldn't find any lines that differ, just removed the if